src/test/java/org/springframework/data/couchbase/util/JavaIntegrationTests.java

```diff
@@ -107,6 +107,7 @@ public static void beforeAll() {
 		ApplicationContext ac = new AnnotationConfigApplicationContext(Config.class);
 		couchbaseTemplate = (CouchbaseTemplate) ac.getBean(COUCHBASE_TEMPLATE);
 		reactiveCouchbaseTemplate = (ReactiveCouchbaseTemplate) ac.getBean(REACTIVE_COUCHBASE_TEMPLATE);
+		System.err.println("JavaIntegrationTests.beforeAll() couchbaseTemplate: " + couchbaseTemplate);
```

Comment:
should these be logged instead? because then you can turn it on off with the logger config, vs these will show up all the time

Reply:
Yes. I'll look for other System.out and change to log.debug